cgi-bin/help.c

```diff
@@ -28,7 +28,7 @@ main(int  argc,				/* I - Number of command-line arguments */
 		*si;			/* Search index */
   help_node_t	*n;			/* Current help node */
   int		i;			/* Looping var */
-  const char	*query;			/* Search query */
+  char		*query = NULL;			/* Search query */
```

Comment:
This can stay const, but do assign to NULL.

Reply:
Updated, thanks!